src/app/win32_service.cc

```diff
@@ -151,8 +152,11 @@ void I2PService::Start(
 
 void I2PService::OnStart(
     DWORD argc,
-    PSTR *argv) {
-  // TODO(unassigned): unused args
+    PSTR * argv) {
+  std::vector<std::string> args{};
+  for (DWORD i=0; i < argc; ++i)
+    args.push_back(argv[i]);
+  Daemon.Configure(args);
```

Comment:
I can't confirm this on windows run-time because of https://github.com/monero-project/meta/issues/238 but this looks logically sound.